python/lsst/ts/ofc/ofc_controller.py

```diff
@@ -318,16 +324,33 @@ def uk_gain(self, filter_name, dof_state):
             Name of the filter.
         dof_state : `numpy.ndarray`
             Optical state in the basis of DOF.
+        sensor_names : `list` of `string`
+            List of sensor names.
 
         Returns
         -------
         uk : `numpy.ndarray`
             Calculated uk in the basis of DOF.
+
+        Raises
+        ------
+        RuntimeError
+            If `sensor_names` is not provided for full array mode instruments.
         """
 
-        return self.gain * self.uk(filter_name, dof_state)
+        if self.ofc_data.name != "lsst" and sensor_names is None:
```

Comment:
I thought the `black` will do:

```
if (self.ofc_data.name != "lsst") and (sensor_names is None):
```

Reply:
I run black and didn't do it, anyway fixed it!